fhir-profile/src/main/java/com/ibm/fhir/profile/ProfileSupport.java

```diff
@@ -203,11 +211,8 @@ public static Binding getBinding(String path) {
     }
 
     public static Map<String, Binding> getBindingMap(String url) {
-        Map<String, Binding> bindingMap = BINDING_CACHE.get(url);
-        if (bindingMap == null) {
-            bindingMap = BINDING_CACHE.computeIfAbsent(url, ProfileSupport::computeBindingMap);
-        }
```

Comment:
@JohnTimm I think the null checks here aren't really needed because `computeIfAbsent` should already handle that case, right?  Anything I'm missing?